DataFormats/L1TGlobal/interface/GlobalAlgBlk.h

```diff
@@ -109,9 +101,12 @@ class GlobalAlgBlk
     //Prescale Column
     int m_preScColumn;
 
+    //Menu UUID
+    int m_menuUUID;
+    int m_firmwareUUID;
    
     std::vector<bool> m_algoDecisionInitial;
-    std::vector<bool> m_algoDecisionPreScaled;
+    std::vector<bool> m_algoDecisionInterm;
```

Comment:
It changes notation to reflect a hardware change.  We requested that initial decision be truly initial decision, the algorithm output with no mask / veto / pre-scale applied.  So now there is initial / final and one intermediate point which I can't recall the exact intermediate stage.